internal/libooniengine/engine.h

```diff
@@ -19,7 +26,53 @@ char *OONIEngineVersion(void);
 /// OONIEngineFreeMemory frees the memory allocated by the engine.
 ///
 /// @param ptr a void pointer refering to the memory to be freed.
-void OONIENgineFreeMemory(void *ptr);
+void OONIEngineFreeMemory(void *ptr);
+
+/// OONIEngineCall starts a new OONITask using the given [req].
+///
+/// @param req A JSON string, owned by the caller, that
+/// contains the configuration for the task to start.  
+///
+/// @return Zero on failure, nonzero on success. If the return value
+/// is nonzero, a task is running. In such a case, the caller is
+/// responsible to eventually dispose of the task using OONIEngineFreeMemory.
```

Comment:
Are you sure we can use `OONIEngineFreeMemory` to dispose of a task? I thought we should be having a dedicated API that removes the corresponding handle from the corresponding map.